metricbeat/module/kvm/dommemstat/dommemstat.go

```diff
@@ -162,7 +162,7 @@ func getDomainMemoryStatName(tag int32) string {
 	case 5:
 		return "available"
 	case 6:
-		return "actualballon"
+		return "actualballoon"
```

Comment:
afaict this is the correct spelling of the kvm thing..

Reply:
@amandahla Can you confirm?

Reply:
Yes, the right spelling is "actualballoon"